arcade/gl/buffer.py

```diff
@@ -66,7 +66,7 @@ def __init__(
         elif reserve > 0:
             self._size = reserve
             # populate the buffer with zero byte values
-            data = (gl.GLubyte * self._size)(0)
+            data = (gl.GLubyte * self._size)()
```

Comment:
@einarf Is this a bug?

Reply:
I also wonder about this one.

Reply:
Passing X positional args initializes the first X items of the array.  The remaining items initialize to `0`.  So passing `0` wasn't hurting, but its presence was confusing.

https://docs.python.org/3/library/ctypes.html#arrays